i18n/en.json

```diff
@@ -17,5 +17,6 @@
 	"ext-whowrotethat-tour-welcome-description": "Activate the the tool from the sidebar. Hover over the text to see the author and their other contributions to this page. Click to get more details.",
 	"ext-whowrotethat-tour-welcome-dismiss": "Got it!",
 	"ext-whowrotethat-revision-added": "$1 added this on $2.",
-	"ext-whowrotethat-revision-attribution": "They have written <strong>$1%</strong> of the page."
+	"ext-whowrotethat-revision-attribution": "They have written <strong>$1%</strong> of the page.",
+	"ext-whowrotethat-revision-username": "User:$1"
```

Comment:
Hmm. `User:` can be gendered in some languages (like Hebrew, and I think Spanish?). Most cases, we'd solve it by adding a {{GENDER}} rule to the translation *but* that rule takes into account the viewer, not the user that's presented. To show the user presented, we'd need to ask the server per user coming in.

Seeing as that's the case, honestly, we should talk to Product and maybe just display the username without the `User:` prefix.

For the moment, I think it's okay to leave that in (I won't consider it a blocker for merging) but it's definitely something we should consider removing or dealing with, as it will look very odd and not consistent with the rest of the wiki in gendered languages.

Reply:
Spoke and approved with product -- do not show `User:` prefix at all.